lib/autoproj/autobuild_extensions/package.rb

```diff
@@ -3,7 +3,7 @@ module AutobuildExtensions
         module Package
             # Tags explicitely added with #add_tag
             attr_reader :added_tags
-
+            attr_reader :internal_dependencies
```

Comment:
I'm moving away from my very personal style of defining accessors for all instance variables. Even if you do add it, I'd prefer having an `add_` accessor for it.

Reply:
Sounds good to me...